torch/fx/experimental/proxy_tensor.py

```diff
@@ -13,6 +13,7 @@
 from torch._dispatch.python import enable_python_dispatcher
 import torch.fx as fx
 from torch.fx.passes.shape_prop import _extract_tensor_metadata
+import functorch._src.config as config
```

Comment:
uh... this is causing circular import... 
functorch imports make_fx, and I cannot import functorch here... 

Any suggestion where I put the use_pre_autograd_decomposition flag?

Reply:
Do we have to flag guard this. Can we not flag guard it lol

Reply:
Another way to keep the flags in functorch is to make it a kwarg to `make_fx` and then have the call sites in functorch pass in the flag